tests/e2e_tests/ext_json_implicit_type_conversion.yml

```diff
@@ -56,13 +56,13 @@ tests:
     current_db: test_db
     query: "SELECT '{\"$numberInt\": \"1\"}' AS s FROM [{}] AS t"
     result:
-      - { 't': { 's': '{"$numberInt": "1"}' } }
+      - { '': { 's': '{"$numberInt": "1"}' } }
```

Comment:
Wow yes, this is a huge difference! Excellent find and fix 💯

It's great, of course, that the implementation was actually correct all along and just the test runner was buggy but still it's obviously so critical to have a correct test framework 🤦‍♀️